x/ecosystem-incentive/keeper/keeper.go

```diff
@@ -0,0 +1,52 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/tendermint/tendermint/libs/log"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	"github.com/UnUniFi/chain/x/ecosystem-incentive/types"
+)
+
+type (
+	Keeper struct {
+		cdc           codec.BinaryCodec
+		storeKey      storetypes.StoreKey
+		memKey        storetypes.StoreKey
```

Comment:
Can you remove `memKey` if it's not used?

Reply:
removed.